src/frontend/src/pages/ChangeRequestDetailPage/OtherChangeRequestsPopupTabs.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import React, { useState } from 'react';
+import { Box, useTheme, Collapse, Tabs, Tab, Typography } from '@mui/material';
+import { ChangeRequest } from 'shared';
+import ChangeRequestDetailCard from '../../components/ChangeRequestDetailCard';
+import { useAllChangeRequests } from '../../hooks/change-requests.hooks';
+import { ExpandLess, ExpandMore } from '@mui/icons-material';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+
+interface OtherChangeRequestsPopupTabsProps {
+  changeRequest: ChangeRequest;
+}
+
+const OtherChangeRequestsPopupTabs: React.FC<OtherChangeRequestsPopupTabsProps> = ({
+  changeRequest
+}: OtherChangeRequestsPopupTabsProps) => {
+  const theme = useTheme();
+  const [tab, setTab] = useState(0);
+  const { data: changeRequests, isError, isLoading, error } = useAllChangeRequests();
+
+  if (isLoading) return <LoadingIndicator />;
+  if (isError) return <ErrorPage error={error} message={error.message} />;
+
+  // the CRs submitted or reviewed by the submitter of this CR
+  const crFromSameUser = changeRequests?.filter(
+    (cr) =>
+      (cr.submitter.userId === changeRequest.submitter.userId || cr.reviewer?.userId === changeRequest.submitter.userId) &&
+      cr.crId !== changeRequest.crId
+  );
+
+  const displayTab = (value: number, title: string) => (
+    <Tab
+      value={value}
+      label={
+        <Typography sx={{ display: 'flex' }}>
+          {title}
+          {tab === value ? <ExpandMore sx={{ pl: 0.5 }} /> : <ExpandLess sx={{ pl: 0.5 }} />}
+        </Typography>
+      }
+      onClick={() => {
+        tab === value && setTab(0);
```

Comment:
for a single body line code in the arrow function, we can remove { } and semicolon, ; to reduce the code. 

In other words, let's replace this onClick to the following: `onClick={() => tab === value && setTab(0)}`